src/atoms/Checkbox/Checkbox.native.js

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useCallback, useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Text, TouchableOpacity, View } from 'react-native';
+import styled, { ThemeContext } from 'styled-components/native';
+import CheckBoxIcon from './CheckboxIcon';
+
+const CheckboxContainer = styled.View`
+  display: flex;
+  flex-direction: row;
+`;
+
+const RippleView = styled(View)(
+  (props) => `
+  width: ${props.width}px;
```

Comment:
Aren't we using a pixel function? @kamleshchandnani

Reply:
we are it's dependent on other PR. My bad I converted this PR to `Ready for review` from `Draft` 😄